### PR TITLE
[#804] Spec: Use “https://” instead of “http://” for any external link

### PR DESCRIPTION
**Pull request by ThomasVitale**
Replace any "http://" occurrence in an external link with "https://".

gh-804

### CODE REVIEW

File: jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc [hunk 1]
```diff
@@ -2,33 +2,33 @@
 == Bibliography
 
 - [[[bib1,1]]] R. Fielding. Architectural Styles and the Design of Network-based Software Architectures. Ph.d
-               dissertation, University of California, Irvine, 2000. See http://roy.gbiv.com/pubs/dissertation/top.htm.
+               dissertation, University of California, Irvine, 2000. See https://roy.gbiv.com/pubs/dissertation/top.htm.
 
-- [[[bib2,2]]] REST Wiki. Web site. See http://rest.blueoxen.net/cgi-bin/wiki.pl.
+- [[[bib2,2]]] REST Wiki. Web site. See https://rest.blueoxen.net/cgi-bin/wiki.pl.
```
Comment (ThomasVitale) [inline jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc]: This link is broken. The host is unknown, it cannot be reached, no matter http or https. What to do then? @mkarg

**Comment by mkarg** [comment]
@ThomasVitale You need to sign-off all your commits.

File: jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc (hunk 1 above)
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc]: Keep it as HTTP. In a seprate discussion we need to find replacement.

**Comment by mkarg** [comment]
@ThomasVitale There still seems to be some problem with your sign-off. Have you used `git commit -s` to produce that?

**Comment by ThomasVitale** [comment]
@mkarg I will try again, sorry for that.

File: jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc (hunk 1 above)
Comment (ThomasVitale) [reply jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc]: Done

**Comment by mkarg** [comment]
@spericas Do we really treat small fixes like this as "spec changed" (so it needs to weeks review) or shall we fast-track it?

**Comment by spericas** [comment]
> @spericas Do we really treat small fixes like this as "spec changed" (so it needs to weeks review) or shall we fast-track it?

We don't, let's fasttrack all small fixes.

**Comment by mkarg** [comment]
Due to the announced fast-track please everybody speak up *now* if there shall be no merge of this PR  tomorrow. :-)

**Pull request merged**